src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -193,11 +193,17 @@ class WorkflowView extends Component {
           div({ style: { textTransform: 'capitalize', display: 'flex', alignItems: 'baseline', marginTop: '0.5rem' } }, [
             'Data Type:',
             h(Select, {
-              clearable: false, searchable: false,
+              clearable: true, searchable: false,
               wrapperStyle: { display: 'inline-block', width: 200, marginLeft: '0.5rem' },
               value: rootEntityType,
-              onChange: ({ value }) => {
-                this.setState(_.set(['modifiedAttributes', 'rootEntityType'], value))
+              onChange: selected => {
+                const value = !!selected ? selected.value : undefined
+
+                if (value === config.rootEntityType) {
+                  this.setState(_.unset(['modifiedAttributes', 'rootEntityType']))
```

Comment:
Unsetting here won't work in combination with the `_.merge` above. What about using `null`, and swapping that out for `undefined` right before we send it?